tests/testdata/workflows/well_swapping/add_template_config.yml

```diff
@@ -1,11 +1,10 @@
 templates:
-  -
-    file: "./templates/welopen.jinja"
+  - file: "./templates/welopen.jinja"
```

Comment:
Is there a special reason for the special file names? `welopen` and `welclose` in place of `wellopen` and `wellclose` ?